internals/core_models.py

```diff
@@ -1096,7 +1094,7 @@ def __init__(self, *args, **kwargs):
 
   @classmethod
   def get_feature_entry(self, feature_id, update_cache=False):
-    KEY = '%s|%s' % (Feature.DEFAULT_CACHE_KEY, feature_id)
```

Comment:
@jrobbins should be this `FeatureEntry.DEFAULT_CACHE_KEY`? I think I have to pass the cache_key in as well

Reply:
Good point.  It should be different because there will be a time period in which we have both models used at the same time.